packages/ceramic-account/src/did.ts

```diff
@@ -0,0 +1,33 @@
+import type { CeramicApi, CeramicDocument } from './types'
+
+class DIDDocument {
+  constructor (public ceramicDoc: CeramicDocument, private _ceramic: CeramicApi) { }
```

Comment:
This class looks quite nice. It might become the main interface for 3ID doctypes in the future!